docs/guides/src/main/operator/basic-deployment.adoc

```diff
@@ -171,4 +171,24 @@ For debugging and development purposes we suggest you to directly connect to the
 kubectl port-forward service/example-kc-service 8443:8443
 ----
 
+==== Accessing the Admin Console
+
+When deploying Keycloak the operator will generate arbitrary initial admin username and password and store those credentials as a Kubernetes basic-auth Secret in the same namespace as the CR.
```

Comment:
```suggestion
When deploying Keycloak, the operator generates an arbitrary initial admin-username and -password and stores those credentials as a Kubernetes basic-auth Secret in the same namespace as the CR.
```

Reply:
partially integrated, in this case, we want to use `username` and `password` as they are the specific fields:
https://kubernetes.io/docs/concepts/configuration/secret/#basic-authentication-secret